hugo/content/tags/css/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: CSS
+description: CSS stands for Cascading Style Sheets· CSS is the language we use to style a Web page.
+link:
```

Comment:
https://www.w3.org/TR/CSS/#css might be a good link. Wikipedia uses it. If I were you I'd ask

Reply:
added the link